lib/query_builder/schema/comparison_filter.ex

```diff
@@ -25,8 +25,8 @@ defmodule EctoShorts.QueryBuilder.Schema.ComparisonFilter do
   end
 
   def build(query, filter_field, filters) when is_map(filters) do
-    Enum.reduce(filters, query, fn ({filter_type, value}, query_acc) ->
-      build_subfield_filter(query_acc, filter_field, filter_type, value)
+    Enum.reduce(filters, query, fn ({filter_type, value}, query) ->
+      convert_to_field_comparison_filter(query, nil, filter_field, filter_type, value)
```

Comment:
I'm wondering if there's a better name here, field_comparison_filter makes me think build isn't comparing fields, maybe it'd be better as `build_schema_field_filters` also to match with naming of build

Reply:
done